src/main/java/edu/ucla/library/services/metadata/MetadataSetter.java

```diff
@@ -328,6 +338,16 @@ private void addMetadata(final String... aRow) throws FileNotFoundException, FfP
             final FFmpegFormat format = probeResult.getFormat();
 
             aRow[aRow.length - DURATION_OFFSET] = String.valueOf(format.duration);
+            if (myCsvHeaders.hasFormatExtentIndex() && aRow[myCsvHeaders.getFormatExtentIndex()].trim().equals("")) {
```

Comment:
I was under the impression that the columns we create are only the ones that the program creates (we never expect users to create them). This would be different, I think, from the columns we expect the DLCS exporter to have created (or a hypothetical user). It's an interesting question though. I haven't looked at all the cases to see if my impression holds up.

Reply:
I think if I were a user then I would be surprised that this program creates certain columns and not others. I think we should be consistent with whatever choice we make (within each program, at least, but also preferably for all of them, IMO). So since this program creates all the other columns, I think it should create this one too.

Reply:
Heh, just realizing this particular example is the exception to my impression. It's creating a column that the DLCS export usually does. Today is my Monday. I might be slower than normal.

Reply:
Creating _all_ columns that don't already exist might be a good thing to do when/if we externalize the CSV handling into its own library (as you, @markmatney, suggested not too long ago).

Reply:
1) I'm not actually creating the column--just making sure it exists before trying to populate it. 2) I'm checking in wit Kristian whether column is standard part of DLCS export, and whether metagetter should create column if it's not present.

Reply:
Yes, thanks for the correction, @DRickard